cves/kernel/CVE-2018-18710.yml

```diff
@@ -448,7 +467,11 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    There is another cast from unsigned long to int which causes
```

Comment:
I'm not sure this fully answers the question. I'd try to comment more on the mindset that allowed this mistake

Reply:
Rephrased